### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by LeahWinters**
# PR Description
Please include a summary of the change:

Melissa and I added test coverage for the Header component and the Login component to ensure the components are rendering correctly. Also added tests to mock out any action the user my take, to ensure the application is working as expected.

Commit Emojis:
:bar_chart: new tests

If it is a new feature, describe the feature and the iteration of the project spec it addresses.

Note any questions or struggles as well.

Currently stuck with an error pointed at an await waitFor. We tried updating all of our testing packages and we researched extensively on how to resolve the problem, but have not come to a solution yet.  We just reached out to our instructor in hopes to get the test passing.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Tests (non-breaking change which adds testing)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [X] My code follows the Turing style guidelines
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by st0nebreaker** [comment]
Thanks for getting so much testing done!

File: src/Header/Header.test.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Header from './Header';
+import '@testing-library/jest-dom';
+import { render, fireEvent } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Header', () => {
+  it('Should only display the title of the application when user is not logged in', () => {
```
Comment (asodemann18) [inline src/Header/Header.test.tsx:8]: Looks good!

File: src/Login/Login.test.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    const over21Button = getByLabelText('over-21-button');
+
+    fireEvent.click(over21Button);
+
+    expect(mockVerifyUser).toHaveBeenCalledTimes(0);
+  });
+
+  it('User should be able to login if they fill out the input and click the over 21 button', () => {
+    const mockVerifyUser = jest.fn();
```
Comment (asodemann18) [inline src/Login/Login.test.tsx:56]: Nice mocked function

**Review by asodemann18 — APPROVED** [review]
Everything looks good!

**Pull request merged**